cmd/kube-apiserver/app/options/options.go

```diff
@@ -79,6 +79,9 @@ type ServerRunOptions struct {
 	MasterCount            int
 	EndpointReconcilerType string
 
+	LeaseDurationSeconds      int
```

Comment:
I know it makes everything more verbose, but I think it'll be more understandable if we say "IdentityLease" everywhere :/

Reply:
That makes sense. `kube-apiserver --identity-lease-duration-seconds` is more understandable.